doc/manuals/installation_and_administration_manual/gui.md

```diff
@@ -0,0 +1,431 @@
+
+First of all, log into the server machine as any other sudoer user and create the `cosmos-sudo` user:
+
+    $ sudo useradd cosmos-sudo
+
+Do not add a password to the `cosmos-sudo` user since only the ssh keypair will be used for authentication.
+
+Add this user to the sudoers group:
+
+    $ visudo
+
+Add the following line to the appropriate section:
+
+    cosmos  ALL=(ALL)       ALL
+
+Now, log as `cosmos-sudo` and create the ssh keypair; when prompted for the passphrase, leave it empty; use the default `id_rsa` name for the private key (`id_rsa.pub` will be the public counterpart):
+
+    $ su - cosmos-sudo
+    $ ssh-keygen
+
+Despite the empty passphrase, it is necessary to remove it (it exists, but it is empty):
+
+    $ openssl rsa -in ./ssh/id_rsa -out ./ssh/id_rsa2
+
+Then, change the permissions of the private key, the default ones are too open:
+
+    $ chmod 600 ./ssh/id_rsa2
+
+The private key must be copied somewhere the GUI running in the client machine may found it within the `cosmos-gui` user account, let's say `fiware-cosmos/cosmos-gui/conf/`:
+
+    $ scp ./ssh/id_rsa2 cosmos-gui@client_vm:/home/cosmos-gui/fiware-cosmos/cosmos-gui/conf/
+
+Copy the public key to the `authorized_keys` file as well; this file is read by ssh when authenticating as the `cosmos-sudo` user:
+
+    $ cat /home/cosmos-sudo/.ssh/id_rsa.pub >> /home/cosmos-sudo/.ssh/authorized_keys
+
+Finally, you can check the access from the client machine:
+
+    $ su - cosmos-gui
+    $ ssh -i conf/id_rsa2 cosmos@server_vm
+
+[Top](#top)
+
+###<a name="section6.2"></a>Annex B: Creating a self-signed certificate
+
+First of all, create a private key; it may not be necessary if you already have one:
+
+    $ openssl genrsa -out private-key.pem 1024
+
+Second, create a Certificate Signing Request (CSR) using the private key:
+
+    $ openssl req -new -key private-key.pem -out csr.pem
+
+Finally, create the self-signed certificate:
+
+    $ openssl x509 -req -in csr.pem -signkey private-key.pem -out public-cert.pem -days 1000
+
+Please observe a duration of 1000 days for the certificate has been specified.
+
+[Top](#top)
+
+###<a name="section6.3"></a>Annex C: Binding the GUI to a port under TCP/1024
+This GUI may run in any port the user configures. Nevertheless, most of the Unix-like systems avoid non sudoer users to bind applications to ports under 1024. Since the `cosmos-gui` user is not a sudoer one, you won't be able to run the GUI in the typical TCP/443 port, for instance.
+
+In order to solve this, there are several possibilities. One of them is setting the `cap_net_bind_service` <i>capability</i>:
+
+    $ setcap cap_net_bind_service=+ep /path/to/cosmos-gui
+
+Nevertheless, the above shows some caveats, for instance, it is only valid for kernels over 2.6.24, and Linux will disable `LD_LIBRARY_PATH` on any program that has elevated privileges like `setcap` or `suid`.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/214/commits/a37133d55f651e5407b3eb87fb9192bc1a85c89e